### ISSUE TITLE
post_install fails with installer_type: pkg on OSX

### ISSUE THREAD

**Issue by newville** [conda/constructor#241]
This is related to #220.

I would like to be able to use a `post_install` script when making on OSK pkg installer.  FWIW, I'm using `post_install` script to run a freshly installed python program to create a desktop App using https://github.com/newville/pyshortcuts  This works well for all other `constructor` products (Unix shell, Windows installer), but not for OSX PKG.

It appears that a user-specified `post_install` script listed in `construct.yaml` is simply not included in any of the "packages" used by `pkgbuild`, and so is not run at the end of the installation.   

An example  showing the problem would have `construct.yaml` of

```
name: osxpkgtest
version: 1

channels:
    - http://repo.anaconda.com/pkgs/main/

attempt_hardlinks: True

specs:
    - python
    - conda
    - openssl
    - asn1crypto
    - readline
    - setuptools
    - numpy

post_install: post_install.sh   [unix]
installer_type: pkg         [osx]
```

and `post_install.sh`  of
```
#!/bin/bash
echo '## Hello from Post_install script ' > $HOME/postinstall.txt
printenv >> $HOME/postinstall.txt
```

This will create a pkg installer and that installer will run to completeion, but the user-supplied `post_install` script will not be included or run.

I believe there should be another "package" built to run the users `post_install` script.   I will propose a change in `osxpkg.py`

**Comment by nehaljwani** [comment]
Happy to look at a PR, if you have one!

**Comment by mcg1969** [comment]
Fixed, by you! in #242. Thank you!
